src/scss/main.scss

```diff
@@ -52,22 +52,26 @@
 //* Buttons *//
 
 a.button,
-input.button {
-  position: relative;
+input.button,
```

Comment:
There shouldn't be `input`s with a class of "button". Instead use an actual button or `type="button"` (and selector `input[type="button"]`